modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CppPistacheServerCodegen.java

```diff
@@ -285,7 +285,7 @@ public String apiFilename(String templateName, String tag) {
 
     @Override
     public String toApiFilename(String name) {
-        return  modelNamePrefix + camelize(name) + "Api";
+        return  modelNamePrefix + toModelName(name) + "Api";
```

Comment:
We use toModelName in toApiFilename? Wouldn't a toApiName make more sense and make the code more readable?

Reply:
You are right, it is the correct way. the super class `AbstractCppCodegen` does it correctly already
```C++
    public String toApiName(String type)
```

Updated